Index: twidere/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- twidere/build.gradle	(date 1473397693000)
+++ twidere/build.gradle	(revision )
@@ -6,7 +6,7 @@
 apply plugin: 'androidsvgdrawable'
 
 // START Non-FOSS component
-apply plugin: 'io.fabric'
+//apply plugin: 'io.fabric'
 // END Non-FOSS component
 
 buildscript {
@@ -15,13 +15,13 @@
     repositories {
         jcenter()
         // START Non-FOSS component
-        maven { url 'https://maven.fabric.io/public' }
+//        maven { url 'https://maven.fabric.io/public' }
         // END Non-FOSS component
     }
 
     dependencies {
         // START Non-FOSS component
-        classpath 'io.fabric.tools:gradle:1.21.7'
+//        classpath 'io.fabric.tools:gradle:1.21.7'
         // END Non-FOSS component
 
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
@@ -51,7 +51,7 @@
 
     productFlavors {
         // START Non-FOSS component
-        google {}
+//        google {}
         // END Non-FOSS component
         fdroid {}
     }
@@ -81,7 +81,7 @@
     mavenLocal()
     maven { url 'https://s3.amazonaws.com/repo.commonsware.com' }
     // START Non-FOSS component
-    maven { url 'https://maven.fabric.io/public' }
+//    maven { url 'https://maven.fabric.io/public' }
     // END Non-FOSS component
     flatDir { dirs "$projectDir/lib" }
 }
@@ -102,11 +102,11 @@
     compile project(':twidere.component.nyan')
 
     // START Non-FOSS component
-    googleCompile 'com.google.android.gms:play-services-maps:9.4.0'
-    googleCompile 'com.google.android.gms:play-services-auth:9.4.0'
-    googleCompile 'com.google.maps.android:android-maps-utils:0.4.4'
-    googleCompile('com.crashlytics.sdk.android:crashlytics:2.6.2@aar') { transitive = true }
-    googleCompile ':YouTubeAndroidPlayerApi:1.2.2@jar'
+//    googleCompile 'com.google.android.gms:play-services-maps:9.4.0'
+//    googleCompile 'com.google.android.gms:play-services-auth:9.4.0'
+//    googleCompile 'com.google.maps.android:android-maps-utils:0.4.4'
+//    googleCompile('com.crashlytics.sdk.android:crashlytics:2.6.2@aar') { transitive = true }
+//    googleCompile ':YouTubeAndroidPlayerApi:1.2.2@jar'
     // END Non-FOSS component
 
     fdroidCompile 'org.osmdroid:osmdroid-android:5.4:release@aar'
@@ -209,5 +209,5 @@
 }
 
 // START Non-FOSS component
-apply plugin: 'com.google.gms.google-services'
+//apply plugin: 'com.google.gms.google-services'
 // END Non-FOSS component
\ No newline at end of file
