Index: twidere/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- twidere/build.gradle	(date 1485067986000)
+++ twidere/build.gradle	(revision )
@@ -7,20 +7,20 @@
 apply plugin: 'androidsvgdrawable'
 
 // START Non-FOSS component
-apply plugin: 'io.fabric'
+//apply plugin: 'io.fabric'
 // END Non-FOSS component
 
 buildscript {
     repositories {
         jcenter()
         // START Non-FOSS component
-        maven { url 'https://maven.fabric.io/public' }
+//        maven { url 'https://maven.fabric.io/public' }
         // END Non-FOSS component
     }
 
     dependencies {
         // START Non-FOSS component
-        classpath "io.fabric.tools:gradle:$fabric_plugin_version"
+//        classpath "io.fabric.tools:gradle:$fabric_plugin_version"
         // END Non-FOSS component
 
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
@@ -50,7 +50,7 @@
 
     productFlavors {
         // START Non-FOSS component
-        google {}
+//        google {}
         // END Non-FOSS component
         fdroid {}
     }
@@ -80,7 +80,7 @@
     mavenLocal()
     maven { url 'https://s3.amazonaws.com/repo.commonsware.com' }
     // START Non-FOSS component
-    maven { url 'https://maven.fabric.io/public' }
+//    maven { url 'https://maven.fabric.io/public' }
     // END Non-FOSS component
     flatDir { dirs "$projectDir/lib" }
 }
@@ -96,18 +96,18 @@
     compile project(':twidere.component.nyan')
 
     // START Non-FOSS component
-    googleCompile "com.google.android.gms:play-services-maps:$play_services_version"
-    googleCompile "com.google.android.gms:play-services-auth:$play_services_version"
-    googleCompile 'com.google.maps.android:android-maps-utils:0.4.4'
-    googleCompile("com.crashlytics.sdk.android:crashlytics:$crashlyrics_version@aar") {
-        transitive = true
-    }
-    googleCompile 'com.anjlab.android.iab.v3:library:1.0.38'
-    googleCompile "com.dropbox.core:dropbox-core-sdk:$dropbox_core_sdk_version"
-    googleCompile ':YouTubeAndroidPlayerApi:1.2.2@jar'
-    googleCompile("com.google.apis:google-api-services-drive:$google_api_drive_version") {
-        exclude group: 'org.apache.httpcomponents'
-    }
+//    googleCompile "com.google.android.gms:play-services-maps:$play_services_version"
+//    googleCompile "com.google.android.gms:play-services-auth:$play_services_version"
+//    googleCompile 'com.google.maps.android:android-maps-utils:0.4.4'
+//    googleCompile("com.crashlytics.sdk.android:crashlytics:$crashlyrics_version@aar") {
+//        transitive = true
+//    }
+//    googleCompile 'com.anjlab.android.iab.v3:library:1.0.38'
+//    googleCompile "com.dropbox.core:dropbox-core-sdk:$dropbox_core_sdk_version"
+//    googleCompile ':YouTubeAndroidPlayerApi:1.2.2@jar'
+//    googleCompile("com.google.apis:google-api-services-drive:$google_api_drive_version") {
+//        exclude group: 'org.apache.httpcomponents'
+//    }
     // END Non-FOSS component
 
     fdroidCompile 'org.osmdroid:osmdroid-android:5.5:release@aar'
@@ -218,5 +218,5 @@
 }
 
 // START Non-FOSS component
-apply plugin: 'com.google.gms.google-services'
+//apply plugin: 'com.google.gms.google-services'
 // END Non-FOSS component
\ No newline at end of file
