Index: twidere/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- twidere/build.gradle	(date 1504708582000)
+++ twidere/build.gradle	(revision )
@@ -20,7 +20,7 @@
         }
 
         // START Non-FOSS component
-        classpath "com.google.gms:google-services:${pluginVersions['PlayServices']}"
+//        classpath "com.google.gms:google-services:${pluginVersions['PlayServices']}"
         // END Non-FOSS component
     }
 }
@@ -57,9 +57,9 @@
 
     productFlavors {
         // START Non-FOSS component
-        google {
-            dimension 'channel'
-        }
+//        google {
+//            dimension 'channel'
+//        }
         // END Non-FOSS component
         fdroid {
             dimension 'channel'
@@ -139,14 +139,14 @@
     implementation project(':twidere.component.nyan')
 
     // START Non-FOSS component
-    googleImplementation "com.google.android.gms:play-services-maps:${libVersions['PlayServices']}"
-    googleImplementation "com.google.android.gms:play-services-auth:${libVersions['PlayServices']}"
-    googleImplementation "com.google.maps.android:android-maps-utils:${libVersions['MapsUtils']}"
-    googleImplementation "com.anjlab.android.iab.v3:library:${libVersions['IABv3']}"
-    googleImplementation "com.dropbox.core:dropbox-core-sdk:${libVersions['DropboxCoreSdk']}"
-    googleImplementation("com.google.apis:google-api-services-drive:${libVersions['GoogleDriveApi']}") {
-        exclude group: 'org.apache.httpcomponents'
-    }
+//    googleImplementation "com.google.android.gms:play-services-maps:${libVersions['PlayServices']}"
+//    googleImplementation "com.google.android.gms:play-services-auth:${libVersions['PlayServices']}"
+//    googleImplementation "com.google.maps.android:android-maps-utils:${libVersions['MapsUtils']}"
+//    googleImplementation "com.anjlab.android.iab.v3:library:${libVersions['IABv3']}"
+//    googleImplementation "com.dropbox.core:dropbox-core-sdk:${libVersions['DropboxCoreSdk']}"
+//    googleImplementation("com.google.apis:google-api-services-drive:${libVersions['GoogleDriveApi']}") {
+//        exclude group: 'org.apache.httpcomponents'
+//    }
     // END Non-FOSS component
 
     fdroidImplementation "org.osmdroid:osmdroid-android:${libVersions['OSMDroid']}"
