Index: twidere/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- twidere/build.gradle	(date 1455348383000)
+++ twidere/build.gradle	(revision )
@@ -3,17 +3,17 @@
 import java.text.SimpleDateFormat
 
 apply plugin: 'com.android.application'
-apply plugin: 'io.fabric'
+//apply plugin: 'io.fabric'
 apply plugin: 'com.neenbedankt.android-apt'
 apply plugin: 'androidsvgdrawable'
 
 buildscript {
     repositories {
-        maven { url 'https://maven.fabric.io/public' }
+//        maven { url 'https://maven.fabric.io/public' }
     }
 
     dependencies {
-        classpath 'io.fabric.tools:gradle:1.21.2'
+//        classpath 'io.fabric.tools:gradle:1.21.2'
     }
 }
 
@@ -38,7 +38,7 @@
         targetCompatibility JavaVersion.VERSION_1_7
     }
     productFlavors {
-        google {}
+//        google {}
         fdroid {}
     }
     buildTypes {
@@ -67,7 +67,7 @@
 
 repositories {
     maven { url 'https://s3.amazonaws.com/repo.commonsware.com' }
-    maven { url 'https://maven.fabric.io/public' }
+//    maven { url 'https://maven.fabric.io/public' }
     flatDir { dirs "$projectDir/lib" }
 }
 
@@ -123,10 +123,10 @@
     compile project(':twidere.component.common')
     compile project(':twidere.component.nyan')
 
-    googleCompile 'com.google.android.gms:play-services-maps:8.4.0'
-    googleCompile 'com.google.maps.android:android-maps-utils:0.4'
-    googleCompile('com.crashlytics.sdk.android:crashlytics:2.5.5@aar') { transitive = true }
-    googleCompile ':YouTubeAndroidPlayerApi:1.2.2@jar'
+//    googleCompile 'com.google.android.gms:play-services-maps:8.4.0'
+//    googleCompile 'com.google.maps.android:android-maps-utils:0.4'
+//    googleCompile('com.crashlytics.sdk.android:crashlytics:2.5.5@aar') { transitive = true }
+//    googleCompile ':YouTubeAndroidPlayerApi:1.2.2@jar'
 
     fdroidCompile 'org.osmdroid:osmdroid-android:5.1'
 
\ No newline at end of file
